recipe/meta.yaml

```diff
@@ -14,20 +13,24 @@ source:
   sha256: d61c3ef9b09d55b42e038f8d767fa483ebbdec2a9c7172b1b0ccda0ae0016ec9
 
 build:
-  merge_build_host: true  # [win]
-  number: 1
+  number: 2
   rpaths:
     - lib/R/lib/
     - lib/
 
 requirements:
   build:
     - cross-r-base {{ r_base }}  # [build_platform != target_platform]
-    - posix                # [win]
     - {{ compiler('fortran') }}        # [unix]
     - {{ compiler('m2w64_fortran') }}  # [win]
     - {{ compiler('c') }}        # [unix]
+    - {{ stdlib("c") }}          # [unix]
     - {{ compiler('m2w64_c') }}  # [win]
+    - {{ stdlib("m2w64_c") }}    # [win]
+    - {{ posix }}make
+    - {{ posix }}coreutils       # [win]
+    - {{ posix }}sed             # [win]
+    - {{ posix }}zip             # [win]
```

Comment:
@isuruf, I added these because the logs indicated they are missing.
(They were probably brought in via `posix` before.)
And only now I realized that `conda-build`'s CRAN skeleton would add these if `make` is needed.
Now, the skeleton also adds `m2-filesystem` explicitly.
Does it still make sense to add it explicitly?
(Currently, it's indirectly pulled in through `m2-make`/`m2-sed` and their dependency on `m2-bash`.)

Reply:
I don't mind either way. I manually added `m2-filesystem` as a dep to `m2-bash` at https://github.com/conda-forge/m2-binary-packages-feedstock/blob/main/recipe/msys2-pkgs.py#L155-L157 so that we get the bashrc files, fstab fix etc.